model/Service/Classes/SoftwareService.md

```diff
@@ -19,7 +19,7 @@ SoftwareService represents software being licensed, delivered and accessed onlin
   - type: AuthenticationProtocolType
   - minCount: 0
 - serviceHostingCountry
-  - type: CountryCodeAlpha3
+  - type: Core/CountryCodeAlpha3
```

Comment:
```suggestion
  - type: /Core/CountryCodeAlpha3
```

Need the leading slash.

See for example: https://github.com/spdx/spdx-3-model/blob/8caa3fb87d53b356b211a64aca5f6901e14be441/model/Software/Classes/File.md?plain=1#L22-L27